.github/workflows/ci.yml

```diff
@@ -71,7 +67,7 @@ jobs:
             node_modules
             packages/*/node_modules
             packages/*/.vscode-test
-          key: ${{ runner.os }}-${{ matrix.node-version }}-node-modules-${{ hashFiles('yarn.lock')}}
+          key: ${{ runner.os }}-${{ matrix.node-version }}-node-modules-1-${{ hashFiles('yarn.lock')}}
```

Comment:
Why was it necessary to rev this?

Reply:
windows cache wasn't pulling in new dependencies so manually invalidated